enterprise/README.md

```diff
@@ -17,8 +17,26 @@ admin having to ssh into the machine.
 
 To deploy this template from the command line, please make a copy of
 `connectdeploy.sample.template.json` and rename as `azuredeploy.template.json`.
-Update the values in `azuredeploy.template.json` to your deployment values, then
-run the following commands:
+
+Run the `keyvault.sh` to upload your private key and corresponding .crt file to
```

Comment:
We should say at the beginning, before you start you should have ... and list domain set up, the certificates from an authority, etc.